### PR TITLE
Added danish language

### PR DESCRIPTION
**Pull request by MikkelDK**

### CODE REVIEW

**Comment by FlowingSPDG** [comment]
@MikkelDK Thanks for contributing!
One thing, I need to add language selection for top page.
Can you add Danish menu here as what EN/JP did?
https://github.com/FlowingSPDG/get5-web-go/blob/bb12e7a568a036511eec7be54226a96a12a365b8/web/src/App.vue#L36-L39

**Comment by MikkelDK** [comment]
There. ^^

File: web/src/translations/translations.json [hunk 1]
```diff
@@ -217,5 +217,114 @@
...
+          "UnpauseMatch": "Unpause match",
+          "AddPlayerToTeam1": "Føj spiller til team1",
+          "AddPlayerToTeam2": "Føj spiller til team2",
+          "AddPlayerToSpec": "Føj spiller til tilskuerliste",
+          "SendRcon": "Send rcon kommado",
+          "LoadBackupFile": "Load a backup file",
+          "CancelMatch": "Annuller kamp",
+          "MatchHasBeenCancelled": "Denne kamp er annulleret.",
+          "MatchForfeitedBy": "Denne kamp blev fortabt af {team}.",
+          "MatchPendingStart": "Denne kamp afventer start."
```
Comment (FlowingSPDG) [inline web/src/translations/translations.json]: Add translations for "MatchPendingStart", "MessageCancelSuccess", "MessageAddPlayerSuccess", "MessageSendCommandSuccess".
https://github.com/FlowingSPDG/get5-web-go/blob/bb12e7a568a036511eec7be54226a96a12a365b8/web/src/translations/translations.json#L26-L29

File: web/src/translations/translations.json [hunk 2]
```diff
@@ -217,5 +217,114 @@
...
+          "Owner": "Owner",
+          "LoadMore": "Indlæs mere..."
+        },
+        "Metrics": {
+          "Get5Metrics": "Get5 Metrics"
+        },
+        "MyServers": {
+          "NoServersFound": "Ingen servere fundet.",
+          "InUse": "I brug",
+          "Free": "Ledig"
```
Comment (FlowingSPDG) [inline web/src/translations/translations.json]: Add "MessageDeleteSuccess".
https://github.com/FlowingSPDG/get5-web-go/blob/bb12e7a568a036511eec7be54226a96a12a365b8/web/src/translations/translations.json#L63

File: web/src/translations/translations.json [hunk 3]
```diff
@@ -217,5 +217,114 @@
...
+          "FormServerIP": "Server IP",
+          "FormServerPort": "Server Port",
+          "FormRCONPassword": "RCON Adgangskode",
+          "FormPublicTeam": "Public Team",
+          "MessageRegisterSuccess": "Succesfuld registreret server.",
+          "MessageeEditSuccess": "Server er redigeret."
+        },
+        "Team": {
+          "Players": "Spillere",
+          "RecentMatches": "Seneste Kampe"
```
Comment (FlowingSPDG) [inline web/src/translations/translations.json]: Add "MessageDeleteSuccess".
https://github.com/FlowingSPDG/get5-web-go/blob/bb12e7a568a036511eec7be54226a96a12a365b8/web/src/translations/translations.json#L79

Comment (MikkelDK) [reply web/src/translations/translations.json]: Fixed anything now on: https://github.com/FlowingSPDG/get5-web-go/pull/159/commits/298e00f43063729b8c20cba2166e3579bc29eaff

File: web/src/translations/translations.json (hunk 2 above)
Comment (MikkelDK) [reply web/src/translations/translations.json]: Fixed: https://github.com/FlowingSPDG/get5-web-go/pull/159/commits/298e00f43063729b8c20cba2166e3579bc29eaff

File: web/src/translations/translations.json (hunk 1 above)
Comment (MikkelDK) [reply web/src/translations/translations.json]: Fixed!

**Comment by MikkelDK** [comment]
@FlowingSPDG > Fixed anything now on: https://github.com/FlowingSPDG/get5-web-go/pull/159/commits/298e00f43063729b8c20cba2166e3579bc29eaff =D

**Comment by FlowingSPDG** [comment]
Thanks!

**Pull request merged**